Mods/BreathOfTheWild_Intel Fix Flipped Lights/rules.txt

```diff
@@ -0,0 +1,6 @@
+[Definition]
+titleIds = 00050000101C9300,00050000101C9400,00050000101C9500
+name = Fix Flipped Intel Lights"
+path = "The Legend of Zelda: Breath of the Wild/Graphics/Intel/Remove Flipped Intel Lights"

```

Comment:
You should move this path to `The Legend of Zelda: Breath of the Wild/Workarounds/Intel iGPU/Remove Flipped Intel Lights`.

Reply:
Description updated 

description = This workaround targets users of Intel iGPU drivers 23.xx and older experiencing flipped lighting issues. These drivers may offer better performance but have graphical glitches. This pack aims to fix the graphical issues so the older drivers create visuals similar to the new drivers. These older drivers may be found on Intel's site https://downloadcenter.intel.com/download/27484/Graphics-Intel-Graphics-Driver-for-Windows-15-65-.

Reply:
Path updated
path = "The Legend of Zelda: Breath of the Wild/Workarounds/Intel iGPU/Fix Flipped Intel Lights"